src/pal/inc/unixasmmacros.inc

```diff
@@ -18,7 +18,7 @@
 #define LOCAL_LABEL(name) .L##name
 #endif
 
-#if defined(__APPLE__)
+#if defined(__APPLE__) || defined(__aarch64__)
```

Comment:
This looks strange, I would expect the ARM64 to use name and not _name. Are you sure this is correct?

Reply:
Also, can you please use _ARM64_ instead of __aarch64__?

Reply:
this is the error I'm getting:

 98%] Building ASM object src/vm/wks/CMakeFiles/cee_wks.dir/__/arm64/pinvokestubs.S.o
/src/vm/arm64/crthelpers.S: Assembler messages:
/src/vm/arm64/crthelpers.S:18: Error: unexpected characters following instruction at operand 1 -- `b memset@PLT'
/src/vm/arm64/crthelpers.S:30: Error: unexpected characters following instruction at operand 1 -- `b memcpy@PLT'
src/vm/wks/CMakeFiles/cee_wks.dir/build.make:3023: recipe for target 'src/vm/wks/CMakeFiles/cee_wks.dir/__/arm64/crthelpers.S.o' failed

Reply:
with aarch64 gcc 7.3

Reply:
I am sorry for not being clear. I understand the @PLT doesn't work, but I was thinking that we should be using "name" instead of "_name".